### DISCUSSION TITLE
Managing Gists with Positron - availability of extensions or different approach?

### DISCUSSION THREAD

**Discussion by rkrug** [posit-dev/positron#4592] [Q&A]
Hi
I am looking for a good way to see, upload, delete etc of Gists. In VSCode there exists a few extensions to do that, but I can't find any for Positron - is that a licensing issue or am I missing something?

Are there other approaches / extensions to manage Gists which  am missing?

Cheers,

Rainer

**Comment by DavisVaughan** [comment]
What extensions in particular? Generally you can ask them to cross list on OpenVSX as well, the OpenVSX ecosystem makes it quite easy to do so IIUC

**Comment by rkrug** [comment]
I was looking at the extension Gist (https://marketplace.visualstudio.com/items?itemName=kenhowardpdx.vscode-gist). I was not aware that it has to be done by the maintainer.

Thanks.

**Comment by rkrug** [comment]
But I installed it for now from the `.vslx` file.
